eth/vm/forks/muir_glacier/opcodes.py

```diff
@@ -0,0 +1,6 @@
+from eth.vm.forks.istanbul.opcodes import (
+    ISTANBUL_OPCODES,
+)
+
+
+MUIR_GLACIER_OPCODES = ISTANBUL_OPCODES
```

Comment:
Maybe make a copy to be sure mutation to one doesn't mutate the other?

Reply:
Good point :+1: